tests/kas.spec.ts

```diff
@@ -80,19 +80,19 @@ test('test Kafka list filtered by status', async ({ page }) => {
   await expect(page.getByText('No results found')).toHaveCount(1);
 
   await filterByStatus(page, 'Creating');
-  await expect(page.getByText(testInstanceName)).toBeTruthy();
```

Comment:
revert all changes in this file please

Reply:
![image](https://user-images.githubusercontent.com/100594859/203090001-f9348d78-1171-4d18-aa53-76da7649ee76.png)

Reply:
Please open an issue to discuss if/how we use a linter in this repo, totally up for it, but it needs to be separated from the current PR since it has a different scope.